test/java/build.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="CVC4" default="test" basedir=".">
+  <property name="class-dir" value="classes" />
+  <path id="classpath">
+    <fileset dir="local-lib" includes="**/*.jar" />
```

Comment:
This line is currently causing `master` to fail

Reply:
Any idea on how to fix it? I tried a quick and dirty fix https://github.com/4tXJ7f/CVC4/commit/49340c05764561da0a540d44cfa6092e1af2cc79 but the Bitvector test fails:

```
test:
    [junit] Running BitVectors
    [junit] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.015 sec

BUILD FAILED
/home/travis/build/4tXJ7f/CVC4/test/java/build.xml:23: Test BitVectors failed
```

Reply:
It needs hamcrest in addition to junit.
Both of these are included in `ant-optional` and should be located in `/usr/share/java`

Waiting to hear from Travis about https://github.com/pathawks/CVC4/commit/f501d0406b19adea65400f5222477afc0951a8e9

Reply:
That doesn't work.
https://travis-ci.org/pathawks/CVC4/builds/269578955